modules/ROOT/pages/_includes/versions.txt

```diff
@@ -1,11 +1,11 @@
-:neo4j-ogm-version: 3.1.6
-:spring-data-neo4j-version: 5.1.3.RELEASE
-:neo4j-stable-version: 4.0.3
+:neo4j-ogm-version: 3.2.23
+:spring-data-neo4j-version: 6.1.1
+:neo4j-stable-version: 4.2.6
 :bolt-driver-version: 1.7.0
```

Comment:
not sure what bolt driver refers to

Reply:
it seems `bolt-driver-version` is only used here: https://github.com/neo4j-contrib/preview/blob/37cd8146d271b05444196cc525884cc5954e1b3c/language-guides/_includes/versions.txt
Do you know what https://github.com/neo4j-contrib/preview is for, @jexp ?